crates/contract/Cargo.toml

```diff
@@ -50,6 +50,7 @@ __abi-generate = ["near-sdk/__abi-generate"]
 
 [dependencies]
 anyhow = { workspace = true }
+assert_matches = { workspace = true }
```

Comment:
please remove this dep, I am aware it is very lean, but the least deps we have in the contract prod the better IMO. The point where it is actually used can be easily swapped